changelogs/fragments/3935-add-gitlab-group-runner.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+    - 'gitlab_runner - allow to register group runner (https://github.com/ansible-collections/community.general/pull/3935).'
+bugfixes:
+    - 'gitlab_runner module : Fix project/group runner CRUD operations'
```

Comment:
```suggestion
    - 'gitlab_runner module - fix project/group runner CRUD operations (https://github.com/ansible-collections/community.general/pull/3935).'
```

Reply:
Of course, sorry. I didn't applied your first review comment on the bugfix changelog, but only on the minor_change changelog.